README.md

```diff
@@ -4,6 +4,10 @@ Python samples for [Google Cloud Platform products][cloud].
 
 [![Build Status][py-2.7-shield]][py-2.7-link] [![Build Status][py-3.6-shield]][py-3.6-link] [![Build Status][py-3.7-shield]][py-3.7-link] [![Build Status][py-3.8-shield]][py-3.8-link] [![Build Status][py-3.9-shield]][py-3.9-link] [![Build Status][py-3.10-shield]][py-3.10-link]
 
+## Google Cloud Samples
+
+To browse ready to use code samples check [Google Cloud Samples](https://cloud.google.com/docs/samples?l=python).
```

Comment:
The wording gives an impression that the experience we'd like the users to have is solely on the Sample Browser. Could we rephrase it so that users can take a look at the Sample Browser, as well as code on the repository as well?

Reply:
Thank you!

Reply:
I will do that